Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM node:15.1.0-alpine3.12
```

Comment:
node:lts-alpine

Reply:
fixed in https://github.com/csivitu/email-service-backend/pull/1/commits/fb9a5871f26e83152b7b633a9e77e8ea63dae1f8